tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md

```diff
@@ -12,11 +12,14 @@ primary_tag: software-product>sap-btp--cloud-foundry-environment
 ## Prerequisites
  - You have registered for a Cloud Foundry [trial account] (hcp-create-trial-account) on SAP Business Technology Platform.
  - [Python] (https://www.python.org/downloads/) version 3.6.x or higher is installed locally. In this tutorial, we use Python **3.9.6**.
- - The [Cloud Foundry command line interface] (https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/4ef907afb1254e8286882a2bdef0edf4.html) is installed locally.
- - You have [npm] (https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed locally.
+ - [cf CLI] (https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/4ef907afb1254e8286882a2bdef0edf4.html) is installed locally.
```

Comment:
CF not cf. It's a product name.

Reply:
Hi Michael,

I like CF better too. But in the official Cloud Foundry documentation, they always write it **cf CLI**:
https://docs.cloudfoundry.org/cf-cli/install-go-cli.html

Same with our (SAP) official documentation:
https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/4ef907afb1254e8286882a2bdef0edf4.html